virtio-devices/src/net.rs

```diff
@@ -293,6 +296,26 @@ impl Net {
         )
     }
 
+    pub fn from_tap_fd(
```

Comment:
As a follow up PR, it could make sense to have a `from_tap_name()` wrapper as well, for the sake of making the `device_manager` virtio-net device creation code path more consistent.